File: TOI-Zero_68/A2/028/028.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int n;
    cin>>n;
    string a,b;
    cin>>a>>b;
    int cnt=0;
    for(int i=0;i<n;i++){
        if((a[i]-'0')+(b[i]-'0')==9){
            cnt++;
        }
    }
    if(cnt==n){
        cout<<"YES"<<endl;
    }
    else{
        cout<<"NO"<<" "<<n-cnt<<endl;
    }
    return 0;
}

File: Resources/Projects/Archive/2023/Competition/Programming/Contest/TOI-Zero_68/A2/019/TOI-Zero_68/A2/028/028.py
n = int(input())
a = input()
b = input()

cnt = 0
for i in range(n):
    if int(a[i]) + int(b[i]) == 9:
        cnt += 1

if cnt == n:
    print("YES")
else:
    print("NO", n - cnt)